recipes/satsmooth/meta.yaml

```diff
@@ -0,0 +1,65 @@
+
+{% set name = "satsmooth" %}
+{% set version = "1.5.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jgrss/satsmooth/archive/refs/tags/v{{ version }}.zip
+  sha256: 344ea9558e6d6f73b6fedf60a47fdda0fe805b2b160feb2cb38e99fd8a6d03be
+
+# build:
+#   noarch: python
+#   script: {{ PYTHON }} -m pip install . -vv
+#   number: 0
+
+
+build:
+  noarch: python
```

Comment:
Since you are compiling code, this should not be `noarch: python`. You should also be able to use the `pip install` version of `script` you have commented out above.